### PR TITLE
Rename configs

### PR DESCRIPTION
**Pull request by ceyonur**

### CODE REVIEW

File: core/vm/contracts.go [hunk 1]
```diff
@@ -164,9 +163,6 @@ func init() {
 		if _, ok := PrecompileAllNativeAddresses[address]; ok {
 			panic(fmt.Errorf("precompile address collides with existing native address: %s", address))
 		}
-		if address == constants.BlackholeAddr {
```
[the remark this answers is not in the document]
Comment (ceyonur) [reply core/vm/contracts.go:167]: hm I actually moved it to registerer but seems lost in stash. added now. good catch.

File: precompile/modules/registerer.go [hunk 2]
```diff
@@ -51,6 +52,10 @@ func RegisterModule(stm Module) error {
 		return fmt.Errorf("address %s not in a reserved range", address)
 	}
 
+	if address == constants.BlackholeAddr {
+		return fmt.Errorf("address %s overlaps with blackhole address", address)
+	}
+
```
Comment (ceyonur) [inline precompile/modules/registerer.go:58]: moved to here @patrick-ogrady @aaronbuchwald

**Comment by darioush** [comment]
Not against this PR, this is my alternative suggestion https://github.com/ava-labs/subnet-evm/pull/520 happy to follow what others think here to move forward.

File: core/vm/contracts.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (aaronbuchwald) [reply core/vm/contracts.go:167]: Can we keep it in both places?

**Comment by aaronbuchwald** [comment]
Closing in favor of https://github.com/ava-labs/subnet-evm/pull/520

**Pull request closed without merging**